BridgeCareApp/BridgeCare/Services/Simulations.cs

```diff
@@ -97,14 +95,35 @@ public int CreateNewSimulation(int networkId, string simulationName, BridgeCareC
                     WEIGHTING = "none",
                     COMMITTED_START = DateTime.Now.Year,
                     COMMITTED_PERIOD = 5,
-                    DATE_LAST_RUN = DateTime.Now
+                    TREATMENTS = new List<TREATMENT>
+                    {
+                        new TREATMENT()
+                        {
+                            TREATMENT1 = "No Treatment",
+                            BEFOREANY = 1,
+                            BEFORESAME = 1,
+                            BUDGET = null,
+                            DESCRIPTION = "Default Treatment",
```

Comment:
Should strings used in this portion example "No Treatment", "Default Treatment" be defined in resource file or as constants? and if required can be reused.

Reply:
Yes but, copied code originally as it was from legacy app, changed quite a bit...but not this.

Reply:
They are all defaults for setting up the basic simulation, including the number of investments and treatments , they are tied into enums for attributes, enums of allowed treatments etc.... The way to do this might be to setup a default simulation static class and somehow initialize all the allowed values by querying the attributes tables...etc.. If it were one or two string constants I agree , make a resource,  I think we need a whole construct to do this becaue of the relationships.

Reply:
my resolution will be to make a card.

Reply:
ok, please add a card, thanks. Marking it resolved for now then.